### PR TITLE
QOL Updates

### PR DESCRIPTION
**Pull request by Fitblip**
This PR does a few things:
- Adds some console formatting print to show attempts, last check time, and a loader emoji thing
    Screenshot 2023-04-17 at 12 57 01 AM
- Add a .gitignore
- Add a Dockerfile/Makefile (happy to remove/tweak these if you feel strongly about it)
- Remove `quil_voucher.hex` (I think this was checked in accidentally)

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,13 @@
+IMAGE_TAG := quillibrium-ceremony-client
```
Comment (CassOnMars) [inline Makefile:1]:
```suggestion
IMAGE_TAG := quilibrium-ceremony-client
```

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+build-docker:
+	docker build -t $(IMAGE_TAG) .
+
+bash:
+	docker run --rm -it $(IMAGE_TAG) bash
+
+participate: build-docker
+	docker run --rm -it -v $(PWD)/vouchers:/vouchers $(IMAGE_TAG) ./ceremony-client "/vouchers/quill-voucher-$(shell date +'%m.%d.%y-%H:%M:%S').hex"
```
Comment (CassOnMars) [inline Makefile:10]:
```suggestion
	docker run --rm -it -v $(PWD)/vouchers:/vouchers $(IMAGE_TAG) ./ceremony-client "/vouchers/quil-voucher-$(shell date +'%m.%d.%y-%H:%M:%S').hex"
```

File: main.go [hunk 3]
```diff
@@ -26,6 +20,31 @@ func main() {
 	ContributeAndGetVoucher()
 }
 
+func WaitForSequencerToBeReady() {
+	spinnerChars := []string{"🌕", "🌖", "🌗", "🌘", "🌑", "🌒", "🌓", "🌔"}
```
Comment (CassOnMars) [inline main.go:24]: You know why.

```suggestion
	spinnerChars := []string{"⣾", "⣽", "⣻", "⢿", "⡿", "⣟", "⣯", "⣷"}
```

File: main.go [hunk 4]
```diff
@@ -26,6 +20,31 @@ func main() {
...
+		status := fmt.Sprintf("[Attempt %d - Last Checked: %s]", attempts, time.Now().String())
+
+		fmt.Printf("\r%s", removeLine)
+		fmt.Printf("%s\n", message+spinnerChars[spinnerIndex])
+		fmt.Printf("  |- %s", status)
+
+		spinnerIndex = (spinnerIndex + 1) % len(spinnerChars)
+		attempts += 1
+
+		time.Sleep(1 * time.Second)
```
Comment (CassOnMars) [inline main.go:40]: Let's not thrash the sequencer or at least make it configurable – some people are running a lot of these

```suggestion
		time.Sleep(5 * time.Second)
```

File: quil_voucher.hex
Comment (CassOnMars) [inline quil_voucher.hex:1]: It was intentional, let's keep it

**Review by CassOnMars — CHANGES_REQUESTED** [review]
see requested changes, otherwise lgtm

File: main.go (hunk 3 above)
Comment (Fitblip) [reply main.go:24]: Lol I don't, but am down with whatever

File: Makefile (hunk 1 above)
Comment (Fitblip) [reply Makefile:1]: lol I speel gud

**Comment by Fitblip** [comment]
Alright, I think that should sort the requested changes!

**Pull request merged**